examples/slack-connect-example/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "slack-connect-example",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "jsonwebtoken": "^9.0.0",
```

Comment:
This should define `"@knocklabs/react": "*"` as a dependency and import components there using an absolute path instead of the relative path. With the relative path, this example would break if moved out of the monorepo.